src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -324,6 +334,23 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
             if (alibi_slopes->get_element_type() == element::f32) {
                 alibi_slopes = std::make_shared<v0::Convert>(alibi_slopes, element::f32);
             }
+
+            // Jais-13b case
+            if (auto alibi_constant =
```

Comment:
what if "alibi" is not Constant?

Reply:
Fixed.
I decided to treat the presence of Abs as the main sign of Jais-like topology, hence the need to multiply Alibi values by -1. If we encounter the Constant, we may do the additional checking of its values and pray :)
In case of failure, we won't multiply by -1.